samples: matter: light_bulb: Add note about content length option (#13695)
nrfconnect/sdk-nrf: samples/matter/light_bulb/README.rst

1. Complete the setup and configuration described in the :ref:`lib_aws_iot` documentation to get the host name, device ID, and certificates used in the connection.
#. Set the :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` and :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC` Kconfig options in the :file:`overlay-aws-iot-integration.conf` file.

> #. Import the certificates to the :file:`light_bulb/src/aws_iot_integration/certs` folder. The certificates will vary in size depending on the method you chose when generating the certificates. Due to this it migth be needed to increase :kconfig:option:`CONFIG_MBEDTLS_SSL_OUT_CONTENT_LEN` to be able to establish a connection.

Reviewer: ```suggestion
#. Import the certificates to the :file:`light_bulb/src/aws_iot_integration/certs` folder. 
   
   The certificates varies in size depending on the method you chose when generating the certificates. 
   Due to this, you might be need to increase the value of the :kconfig:option:`CONFIG_MBEDTLS_SSL_OUT_CONTENT_LEN` option to be able establish a connection.
```
